src/radio/radio-group.jsx

```diff
@@ -39,6 +39,10 @@ class RadioGroup extends Component {
          * radio group的默认值
          */
         defaultValue: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.bool]),
+        /**
+         * 设置标签类型
+         */
+        component: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
```

Comment:
Would you please add unit test to cover the `PropTypes.func` case?

Reply:
added in 3a80fe7